commons-profiler-api/src/main/java/ru/fix/commons/profiler/impl/ProfiledCallImpl.java

```diff
@@ -71,6 +79,29 @@ public void stop(long payload) {
             sharedCounters.getPayloadSum().add(payload);
             sharedCounters.getMaxThroughput().call();
             sharedCounters.getMaxPayloadThroughput().call(payload);
+
+            sharedCounters.getActiveCalls().remove(this);
+            sharedCounters.getActiveCallsCounter().decrement();
```

Comment:
two decrements of same getActiveCallsCounter. Maybe it would be nice to check unit tests and add assert case to cover this new counters